test-case/multiple-pause-resume.sh

```diff
@@ -30,7 +30,7 @@ OPT_NAME['l']='loop'     OPT_DESC['l']='loop count'
 OPT_HAS_ARG['l']=1         OPT_VAL['l']=3
 
 OPT_NAME['c']='count'    OPT_DESC['c']='combine test pipeline count'
-OPT_HAS_ARG['c']=1         OPT_VAL['c']=2
+OPT_HAS_ARG['c']=1         OPT_VAL['c']=1
```

Comment:
This value shows in the `-h`elp message.  But now it's not the real default value anymore. So the help message needs to be fixed.